source/js/translation.js

```diff
@@ -0,0 +1,56 @@
+let wordsList;
+await fetch('./js/wordsList.json')
+  .then((response) => response.json())
+  .then((data) => {
+    wordsList = data;
+  });
+
+export function replaceWords(
+  inputString,
+  dictionary,
+  sourceLanguage,
+  targetLanguage
+) {
+  if (
+    Object.prototype.hasOwnProperty.call(dictionary, sourceLanguage) &&
+    Object.prototype.hasOwnProperty.call(
+      dictionary[sourceLanguage],
+      inputString
+    )
+  ) {
+    const translation = dictionary[sourceLanguage][inputString];
+    if (Object.prototype.hasOwnProperty.call(translation, targetLanguage)) {
+      return translation[targetLanguage];
+    }
+  }
+  return inputString;
+}
+
+// Can't translate the output from Fortune Teller
```

Comment:
yeah we can leave that for now, maybe do some research to see if you can find a free translate api?

Reply:
Shreya said that she'll handle it